ide/coqide/coqide.ml

```diff
@@ -1413,34 +1417,36 @@ let build_ui () =
   menu file_menu [
     item "File" ~label:"_File";
     item "New" ~callback:File.newfile ~stock:`NEW;
-    item "Open" ~callback:(File.load ~parent:w) ~stock:`OPEN;
+    item "Open" ~callback:(File.load ~parent:w) ~stock:`OPEN ~tooltip:"Open file";
     item "Save" ~callback:(File.save ~parent:w) ~stock:`SAVE ~tooltip:"Save current buffer";
-    item "Save as" ~label:"S_ave as" ~stock:`SAVE_AS ~callback:(File.saveas ~parent:w);
+    item "Save as" ~stock:`SAVE_AS ~callback:(File.saveas ~parent:w);
```

Comment:
Underlined but no hot key.  As I say elsewhere, the underline mechanism seems broken and rather pointless.

![image](https://user-images.githubusercontent.com/1253341/230509712-4cbd009c-c99b-4578-8950-1352b8bdb540.png)

Reply:
I think there is no real standard hotkey for this. Intuitively, I would have chosen Alt-S (because “Save as” is just like an alternative way of saving), and Shift-Ctrl-S for “Save all” (because saving all is like an elevated way of saving), but GTK apps apparently use Shift-Ctrl-S for “Save As.” So you would opt for Shift-Ctrl-S? I rarely use the “Save as” feature, I will rather close my editor program and move my file with my file manager (or the command line). If you need it, Alt-F, A can do the trick; to me, it’s enough.

Reply:
It doesn't have to have an accelerator.